README.md

```diff
@@ -3,27 +3,29 @@ You know how handy that 'Visual Studio Developer Command Prompt' is on your loca
 
 ## Usage
 
-```
+```yml
 - name: Add msbuild to PATH
-  uses: microsoft/setup-msbuild@v1.0.0
+  uses: microsoft/setup-msbuild@v1.0.1
 ```
 
 ## Specifying specific versions of Visual Studio
-You may have a situation where your Actions runner has multiple versions of Visual Studio and you need to find a specific version of the tool.  Simply add the `vs-version` input to specify the range of versions to find.  If looking for a specific version, enter that version number twice as a range.
+You may have a situation where your Actions runner has multiple versions of Visual Studio and you need to find a specific version of the tool.  Simply add the `vs-version` input to specify the range of versions to find.  If looking for a specific version, specify the minimum and maximum versions as shown in the example below, which will look for just 16.4.
 
-```
+```yml
 - name: Add msbuild to PATH
-  uses: microsoft/setup-msbuild@v1.0.0
+  uses: microsoft/setup-msbuild@v1.0.1
     with:
-      vs-version: [16.4,16.5]
+      vs-version: '[16.4,16.5)'

```

Comment:
Is there any syntax that allows for open ended versions? Say, if I'm ok with in the 16.x ranges from 16.4 and up?

Reply:
Never mind, I looked closer at the vswhere help. `'[16.0,17.0)'` is what I'm looking for.